Author: Ben Westover <kwestover.kw@gmail.com>
Summary: Prevent setup.py from installing LICENSE and README.md to /usr
----
--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,6 @@
     packages=find_packages(),
     long_description=open("README.md").read(),
     scripts=['mtk','stage2'],
-    data_files = ['LICENSE','README.md'],
     long_description_content_type="text/markdown",
     url='https://github.com/bkerler/mtkclient',
     project_urls={
